lib/components/user/existing-account-display.js

```diff
@@ -1,34 +1,58 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { Button } from 'react-bootstrap'
+import { connect } from 'react-redux'
+
+import { routeTo } from '../../actions/ui'
 
 import StackedPaneDisplay from './stacked-pane-display'
```

Comment:
I don't know where the idea for this stacked pane UI came from, but I don't think it really works here. If something is unclear in the mockups, we should talk through it before building out components to fill the gaps.